cg/server/api.py

```diff
@@ -224,7 +224,7 @@ def samples():
     elif request.args.get("status") == "sequencing":
         samples_q = db.samples_to_sequence()
     else:
-        customer_objs: Optional[models.Customer] = (
+        customer_objs: Optional[Customer] = (
```

Comment:
```suggestion
        customer: Optional[Customer] = (
```

Reply:
This one looks a  bit mixed in the code between a `Optional[List[Customer]]` and Optional[Customer] and I dot want to dive into refactoring this part now. Let's do it when refactoring Customer SQL